index.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en" xml:lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">  
+   
+    <link rel="stylesheet" type="text/css" href="style.css">
```

Comment:
- [ ] Your page is not fitting on the mobile size. **I noticed that `stylemb.css` is not linked in your header**. Could it be the reason for this 🤔 ?
![view](https://user-images.githubusercontent.com/99656687/236624431-b80cf266-b926-477c-a59d-736b186e4add.png)